### PR TITLE
Added documentation on how to use the machine class testing cli

### PR DESCRIPTION
**Pull request by afritzler**

### CODE REVIEW

File: docs/development/local_setup.md [hunk 1]
```diff
@@ -76,6 +76,17 @@ The Machine Controller Manager should now be ready to manage the VMs in your kub
...
+### Testing Machine Classes
+
+To test the creation/deletion of a single instance for one particular machine class you can use the `managevm` cli. The corresponding `INFRASTRUCTURE-machine-class.yaml` and the `INFRASTRUCTURE-secret.yaml` need to be defined upfront. To build and run it
+
+```bash
+go build cmd/machine-controller-manager/cli/managevm.go
+# create machine
+./managevm --secret ~/temp/openstack-secret.yaml --machineclass ~/temp/openstack-machine-class.yaml --classkind openstack --machinename test
+# delete machine
+./managevm --secret ~/temp/openstack-secret.yaml --machineclass ~/temp/openstack-machine-class.yaml --classkind openstack --machinename test --machineid openstack:///REGION/INSTANCE_ID
```
Comment (prashanth26) [inline docs/development/local_setup.md]: _--machineid providername:///REGION/INSTANCE_ID_ would be generic ?

Comment (afritzler) [reply docs/development/local_setup.md]: Good point! I will make the documentation more generic.

**Comment by afritzler** [comment]
Alright, I made the documentation more generic.

**Comment by prashanth26** [comment]
Thanks Andreas.

**Pull request merged**